docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -246,3 +246,8 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 ## CSharp API
 
 To use csharp api for openvino execution provider create a custom nuget package. Follow the instructions [here](../../BUILD.md##build-nuget-packages) to install prerequisites for nuget creation. Once prerequisites are installed follow the instructions to [build openvino](../../BUILD.md#openvino) and add an extra flag `--build_nuget` to create nuget packages. Two nuget packages will be created Microsoft.ML.OnnxRuntime.Managed and Microsoft.ML.OnnxRuntime.Openvino.
+
+## Multi-threading for OpenVINO EP
+
```

Comment:
The details about the implementation is not required to be documented. Actually, even the multi-threaded inference is a default feature of ONNX Runtime and thus nothing may be required from us to re-document this feature. 
So, on second thought, this documentation would not be required at all. 

You can just mention thread safety in the first line like this:
OpenVINO Execution Provider enables **thread-safe** deep learning inference